packages/landing-page/ui.preset.ts

```diff
@@ -1,7 +1,6 @@
-import { fontFamily } from "tailwindcss/defaultTheme";
+import { type Config } from "tailwindcss";
```

Comment:
ah, it's moved to .ts. Nice!

A side note - we have a bit of a challenge with this
- https://github.com/solidjs/solid-start/issues/1693

Because now Tailwind 4 is has been stable released:
https://npmjs.com/package/@tailwindcss/vite

So we better make sure it works with SolidStart

Reply:
I'd really like us to resolve this asap:
- https://github.com/solidjs/solid-start/issues/1693